de.swsnr.pictureoftheday.yaml

```diff
@@ -0,0 +1,42 @@
+id: de.swsnr.pictureoftheday
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: de.swsnr.pictureoftheday
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    # Tell build.rs not to regenerate the UI files from blueprint sources
+    SKIP_BLUEPRINT: "1"
+modules:
+  - name: pictureoftheday
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/swsnr/picture-of-the-day/releases/download/v0.1.0/picture-of-the-day-v0.1.0.tar.zst
+        sha512: "dc2a2d4bbfee020fb2e8263584925d18d17f6a0a3d033d47316c4614ca16038312b3c0c87ccfee2ec1037b02c02894f5dfe6eda27e2e7f8a8749cbd58745bca2"
+      - type: archive
+        url: https://github.com/swsnr/picture-of-the-day/releases/download/v0.1.0/vendor.tar.zst
+        sha512: "a0ba3e33d11c75b3adc7c2b137449032e798231c550cede6d8638b3ccf6a8f3dd18c961166b0f59341c2d5ef0cb90d8efda1802eca65084ef25d99f208220fc3"
+        dest: vendor/
+      - type: inline
+        dest: .cargo/
+        dest-filename: config.toml
+        contents: |
+          [source.crates-io]
+          replace-with = "vendored-sources"
+
+          [source.vendored-sources]
+          directory = "vendor"
+    build-commands:
+      - cargo build --frozen --release --verbose
+      - make DESTPREFIX=/app install
+      - glib-compile-schemas --strict /app/share/glib-2.0/schemas
```

Comment:
this step should be inside makefile although not a blocker.

Reply:
The distro hook can overwrite, you can search meson.builds in GNOME and many projects use a custom python script to handle this or run it directly via custom target.

Reply:
The rule is that if it is needed, the build system should do it.

Reply:
No, but you can fix it later.